source/mir/random/engine/pcg.d

```diff
@@ -246,6 +246,17 @@ public:
         void popFront()() { state = bump(state); }
         /// ditto
         void seed()(Uint seed) { this.__ctor(seed); }
+        /// ditto
+        static if (streamType == stream_t.none || streamType == stream_t.oneseq)
+        @property typeof(this) save()() const
+        // Repeat the condition so it shows up in DDoc.
+        if (streamType == stream_t.none || streamType == stream_t.oneseq)
```

Comment:
the condition duplicates `static if` above

Reply:
I did that because the auto-generated DDoc will include the function without any mention of `static if`. I'll remove the redundant constraint and instead add a note in the documentation about when `save` is available.